conftest.py

```
@@ -58,12 +83,44 @@ def reset_environment():
+        # App not yet loaded, that's fine
+        pass
+
     yield
 
     # Restore environment
     os.environ.clear()
     os.environ.update(old_env)
 
+    # Clear dependency overrides
+    try:
+        import app as app_module
```

Flake8 (7.2.0)

[error] redefinition of unused 'app_module' from line 88

(F811)